pkg/errors/errors.go

```diff
@@ -7,12 +7,11 @@ package errors
 type TrackingError struct {
 	Err            error
 	HTTPStatusCode int
-	Msg            string
 }
 
 // Error returns an error as string.
 func (te *TrackingError) Error() string {
-	return te.Msg
+	return te.Err.Error()
```

Comment:
I would check if the field: `te.Err` is nil before calling `te.Err.Error()` to avoid posible panics.

Reply:
Fixed in commit fb0c5ce88920525bda9128013c3e070dfead5a46